library/src/main/java/com/getstream/sdk/chat/model/Attachment.java

```diff
@@ -294,6 +296,14 @@ public int getProgress() {
         public void setProgress(int progress) {
             this.progress = progress;
         }
+
+        public boolean isFileTooLarge() {
+            return fileTooLarge;
```

Comment:
a file is too large when its size is > than max size, no need to rely on a private boolean for that, just change this into the expression

Reply:
Yeah, exactly, I know it either.
However if we don't use `setter` and `getter` functions for `fileTooLarge` in Config of `Attachment`  we need some messy codes to check file size exceeds the maxim size.

Each time first we have to get file from file path and then have to compare with limit max upload size.

If the part of checking file size is one or two, it wouldn't problem. unfortunately we have to check this in here and there at least 4 times more.

Reply:
cache the result internally then

Reply:
removed.